src/components/Dashboard/Content/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Shimmer from 'components/Shimmer';
+
+interface DashboardContentProps {
+  'data-test-id'?: string;
+}
+
+const DashboardContent = ({ ...attributes }: DashboardContentProps): JSX.Element => {
+  return (
+    <div className="flex flex-col w-full" {...attributes}>
```

Comment:
we just pass the props thru, it's unnecessary to validate
```suggestion
const DashboardContent = (props): JSX.Element => {
  return (
    <div className="flex flex-col w-full" {...props}>
```

Reply:
or simply
```suggestion
export const DashboardDataTestIds {
	content:  'dashboard-content' // or 'dashboard__content'
}

const DashboardContent = (): JSX.Element => {
  return (
    <div className="flex flex-col w-full" data-test-id={DashboardDataTestIds.content}>
```